src/plugins/visualizations/server/plugin.ts

```diff
@@ -77,6 +80,17 @@ export class VisualizationsPlugin
         category: ['visualization'],
         schema: schema.boolean(),
       },
+      [VISUALIZE_DISABLE_BUCKET_AGG_SETTING]: {
+        name: i18n.translate('visualizations.advancedSettings.visualizeDisableBucketAgg', {
+          defaultMessage: 'Disable visualizations bucket aggregation types',
+        }),
+        value: [],
+        description: i18n.translate('visualizations.advancedSettings.visualizeDisableBucketAgg', {
+          defaultMessage: `Deactivates the specified bucket aggregations from visualizations.`,
```

Comment:
I still think it's worth specifying it's a CSV. But I digress.

Reply:
signed the last commit, not sure what was the verdict for the CSV discussion.
Do you want me to set default values? or it's fine as is? Thanks

Reply:
Default values is fine I think. @kgcreative and @keithhc2 seemed to favor without mentioning that it takes a CSV. But this is just nitpicks and could be easily updated.

Reply:
> signed the last commit

Thanks! Rerunning CI.

Reply:
Sorry, I misunderstood the ask before. If the input is a CSV, then I agree that mentioning specifying that it's a CSV would be helpful. So, it should be something like, "A comma-separated list of <a few accepted settings>" and then link off to the docs for the full list?

Reply:
@RoyiSitbon what do you think, PR is in great shape I can do a fast follow on this change if you do not have the availability.

Reply:
Hey, thanks. Made the description clearer.
Now it's more conspicuous that we use CSV as an input.